omero/sysadmins/unix/install-web.txt

```diff
@@ -68,6 +68,7 @@ more specific walk-through listed below.
     install-web/walkthrough/omeroweb-install-centos7-ice3.6
     install-web/walkthrough/omeroweb-install-ubuntu-ice3.6
     install-web/walkthrough/omeroweb-install-osx-ice3.6
+    install-web/walkthrough/omeroweb-install-debian-ice3.6
```

Comment:
Missing the `.. seealso::` entry to make this visible, line :62 

![setting_up_omero_web_ _omero_documentation](https://user-images.githubusercontent.com/2727871/28876221-5ab1de6a-7790-11e7-9c45-5cf369ff986e.png)